src/optimization.jl

```diff
@@ -0,0 +1,44 @@
+#=
+# Note that by default all particles in the system are assumed optimizable.
+# IMPORTANT: Note that we always work in cartesian coordinates.
+=#
+
+export minimize_energy!
+
+
+""" 
+    By default we work in cartesian coordinaes.
+    Note that internally, when optimizing the cartesian positions, atomic units 
+    are used.
```

Comment:
```suggestion
By default we work in cartesian coordinaes.
Note that internally, when optimizing the cartesian positions, atomic units 
are used.
```

Reply:
Those formatting issues are now fixed. 
I agree on the comments stuff :) will be less verbose from now on.